eox_nelp/admin/course_creators.py

```diff
@@ -0,0 +1,36 @@
+"""courseCreators admin file.
+Contains all the nelped admin models for course_creators.
+classes:
+    nelpCourseCreatorAdmin: EoxNelp CourseCreators admin class.
+"""
+from __future__ import absolute_import
```

Comment:
generally we use this in applications that has support for  python 2 and  3, this is just for python 3 so do you need this for something else ?

Reply:
No, I think no cause we are using python3, I will remove it.

Reply:
Resolved in https://github.com/eduNEXT/eox-nelp/pull/2/commits/a125a7569bbfd536f28897a1a2c94aa41e39a324